pants.toml

```diff
@@ -1,12 +1,13 @@
 
 [GLOBAL]
-pants_version="2.6.0"
+pants_version="2.9.0"
 pythonpath = ["%(buildroot)s/pants_plugins"]
 plugins = [
-   "sendwave-pants-docker@https://github.com/compyman/pants-docker/releases/download/v0.05/sendwave-pants-docker-0.5.tar.gz"
+   "sendwave-pants-docker@https://6-463230233-gh.circle-artifacts.com/0/dist/sendwave-pants-docker-1.0.tar.gz"
```

Comment:
Note to myself: Update this reference once the corresponding PR is merged and released.